metadata-ingestion/tests/integration/vertica/docker-compose.yml

```diff
@@ -4,7 +4,6 @@ services:
     platform: linux/amd64
     environment:
       APP_DB_USER: "dbadmin"
-      APP_DB_PASSWORD: "abc123"
```

Comment:
why is this changed? I think it's better for our integration tests to test using a password

Reply:
We updated the docker image password. Anyhow we have added the password back. It should be good now